content/rooms/v11.md

```diff
@@ -0,0 +1,271 @@
+---
+title: Room Version 11
+type: docs
+weight: 100
+---
+
+This room version builds on [version 10](/rooms/v10) while clarifying redaction
+rules.
+
+## Client considerations
+
+Clients should no longer depend on the `creator` property of `m.room.create` events.
+
+Clients should note that the format of `m.room.redaction` events has been modified
+and look for the `redacts` key under `content` instead of a top-level event property.
+
+Clients should note that the `third_party_invite` key of `m.room.member` events
+is no longer redacted, *but* will only contain the `signed` key after redaction.
+
+Clients which implement the redaction algorithm locally should refer to the
+[redactions](#redactions) section below for a full overview.
+
+## Server implementation components
+
+{{% boxes/warning %}}
+The information contained in this section is strictly for server
+implementors. Applications which use the Client-Server API are generally
+unaffected by the intricacies contained here. The section above
+regarding client considerations is the resource that Client-Server API
+use cases should reference.
+{{% /boxes/warning %}}
```

Comment:
I... am not convinced this is correct. Much of the below is also relevant to clients, as evidenced by the fact the changes are mentioned in the "Client considerations" section.

Can we move the event format, and redaction rule, changes to a "common" section?

Reply:
I believe this is shown on every room version and I just copied and pasted. Will check.

Reply:
I mean sure, and that was probably the right thing to do for some of the early room versions, but that doesn't mean that cutting-and-pasting it every time is still the right thing to do.

Reply:
Weirdly this section seems to mostly duplicate what is written for the clients above though? So I think the idea is that client authors shouldn't need to care about this section since the *above* section provides all the needed info.

Reply:
yeah but the above section *doesn't*. It's currently either a very high-level summary, or (in the case of redactions), a link into this section which it says you shouldn't read if you're a client author.

Reply:
This warning box is intentionally part of the copy/paste template, mandated by documentation style: https://github.com/matrix-org/matrix-spec/blob/main/meta/documentation_style.rst#room-versions . The intention is to prevent client developers from reading "below the fold".

The redactions section actually needs to be duplicated up above, per v9's template (and doc style)